standalone/runtime/tests/test_vault_registry.rs

```diff
@@ -227,6 +229,49 @@ mod withdraw_collateral_test {
             );
         });
     }
+
+    #[test]
+    fn integration_test_vault_registry_withdraw_collateral_respects_custom_thresholds() {
```

Comment:
Question: I think I remember that we discussed that vaults should be able to withdraw their collateral up until the global threshold. Did you guys agree to change that? I'm OK either way.

Reply:
I believe the final conclusion was that the custom threshold should just be used for everything - and if a user wants to override their own threshold, they can batch a transaction to unset it, do whatever they want (including withdrawing collateral, but also e.g. self-issuing), then setting their threshold back.
This simplifies things because we don't need for example two RPCs to let the UI handle collateral withdrawing.